### DISCUSSION TITLE
How to set nerd font symbol as fallback fonts for systemwide settings?

### DISCUSSION THREAD

**Discussion by K4R7IK** [ryanoasis/nerd-fonts#1326] [Q&A] [answered]
Hey, recently I have encountered glyph clipping. After reading among the issues, a solution was to configure nerd font symbol as fallback fonts and use default fonts.
How can I do that? I can't seem to find any article regarding this topic?

**Comment by Finii** [comment, marked as answer]
Some hints are [on our Wiki](https://github.com/ryanoasis/nerd-fonts/wiki/Font-Substitution-&-Fallback).

And there is this [Issue](https://github.com/ryanoasis/nerd-fonts/issues/670) about missing docu, right now unsure if there is content.

**Reply by Finii** [reply]
Hmm, `"SystemDefaultEUDCFont"="Hack Nerd Font Complete Windows Compatible.ttf"` ...
In fact we have the Symbols Only font for font fallback. Hmm. As the Issue says, we have a documentation deficit ;)

**Comment by K4R7IK** [comment]
Thanks, I was able to configure fallback fonts. It works great!
